src/components/nodes/poolLane/index.js

```diff
@@ -3,8 +3,6 @@ import idConfigSettings from '@/components/inspectors/idConfigSettings';
 import nameConfigSettings from '@/components/inspectors/nameConfigSettings';
 
 export const id = 'processmaker-modeler-lane';
-export const minLaneHeight = 100;
-
```

Comment:
This export wasn't actually used anywhere, and the value for the minimum lane height is specified in `src/components/nodes/poolLane/laneSizes.js`.